tasks/security_agent.py

```diff
@@ -57,6 +59,27 @@ def build(
     """
     ldflags, gcflags, env = get_build_flags(ctx, major_version=major_version, python_runtimes='3')
 
+    # generate windows resources
+    if sys.platform == 'win32':
+        windres_target = "pe-x86-64"
+        if arch == "x86":
```

Comment:
I can't imagine we'd ever support this on x86.
Perhaps explicitly disable x86?

Reply:
Will do that later.